server/plugin/command.go

```diff
@@ -532,8 +585,7 @@ func (p *Plugin) ExecuteCommand(c *plugin.Context, args *model.CommandArgs) (*mo
 			text = "Error checking user's permissions"
 			p.API.LogWarn(text, "err", err.Error())
 		case isSysAdmin:
-			githubPluginURL := *p.API.GetConfig().ServiceSettings.SiteURL + "/admin_console/plugins/plugin_github"
-			text = fmt.Sprintf("Before using this plugin, you will need to configure it by filling out the settings in the system console [here](%s). You can learn more about the setup process [here](%s).", githubPluginURL, "https://github.com/mattermost/mattermost-plugin-github#step-3-configure-the-plugin-in-mattermost")
+			text = "Before using this plugin, you will need to configure it by running `/github get-started`"
```

Comment:
should we also say "or, set it up in the system console [here](link to plugin console page)"?  I'm 0/5 on driving the admins through the wizard by omitting this instruction.  @Rina-dsg should we include the alternative way?

Reply:
@aaronrothschild I would push admins to go through the wizard for the first iteration of the wizard in order to test this solution and understand what we can improve.